temporal-sdk/src/main/java/io/temporal/worker/WorkerOptions.java

```diff
@@ -173,6 +177,17 @@ public Builder setActivityPollThreadCount(int activityPollThreadCount) {
       return this;
     }
 
+    /**
+     * If set to true disables activity poller on this worker. Can be useful if you want to handle
+     * workflow tasks only.
```

Comment:
To have workflow tasks only don't call worker.registerActivity. The reason to have this option is to support only local activities by this activity worker. 
Should we rename it to something more explicit? More like `setLocalActivityWorkerOnly`?

Reply:
I don't have strong opinion about naming here as long as default false means normal worker, let me rename to `localActivityWorkerOnly` if you like it better.